### ISSUE TITLE
requires blueprint = ?

### ISSUE THREAD

**Issue by nagyrobi** [xibosignage/xibo#99]
What's the meaning of the blue label "requires blueprint"?

**Comment by knitehawk** [comment]
in laymans terms, it's quite simple.
it means further study is required. (AKA i have no earthly idea, i'll have to think long and hard over a pint or 20 and come up with a diabolical scheme)
these are to be avoided at all costs.
Dan is one of the few contributors to this project, and wew all know what happens after 20 pints or so.
you usually end up singing something dreadful while walking down the street. getting a friendly knock on the head and waking up on the south Indian sea having inexplicably "Volunteered"  to join some unknown country's navy (even more inexplicably because they still row there ships around!)
we need dan, so these are kept to a minimum.


Jk, i couldn't resist

**Comment by alexharrington** [comment]
Exactly that. It means someone needs to draft a full specification for how that would work and how it would be implemented before anything further can happen.
